### PR TITLE
TT-5912 Docs/version

### PR DESCRIPTION
**Pull request by tbuchaillot**
Adding version command to Tyk Sync readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,6 +76,7 @@ Available Commands:
   publish     publish API definitions from a Git repo or file system to a gateway or dashboard
   sync        Synchronise a github repo or file system with a gateway
   update      A brief description of your command
+  version     This command will show the current Tyk-Sync version.
```
Comment (andyo-tyk) [inline README.md:79]: For consistency, can you please delete the full stop `.`?

File: README.md [hunk 2]
```diff
@@ -133,3 +134,13 @@ SYNC Updating Policy: Test policy 1
 
 The command provides output to identify which actions have been taken. If using a Tyk Gateway, the Gateway will be
 automatically hot-reloaded.
+
+
+## Example: Getting the current Tyk Sync version
```
Comment (andyo-tyk) [inline README.md:139]: Perhaps change this to:
`Example: Check the currently installed version of Tyk Sync`

File: README.md [hunk 3]
```diff
@@ -133,3 +134,13 @@ SYNC Updating Policy: Test policy 1
...
+
+In order to get the current Tyk Sync version, we need to run the version command.
```
Comment (andyo-tyk) [inline README.md:141]: Please change this to say:
`To check the current Tyk Sync version, we need to run the version command:`

**Review by andyo-tyk — CHANGES_REQUESTED** [review]

**Comment by tbuchaillot** [comment]
Solved @andyo-tyk

**Review by andyo-tyk — APPROVED** [review]
Thanks!

**Pull request merged**